script/helperFunctions.sh

```diff
@@ -2845,6 +2845,19 @@ function getPrivateKey() {
     fi
   fi
 }
+function isZkEvmNetwork() {
+  # read function arguments into variables
+  local NETWORK="$1"
+
+  # Check if the network exists in networks.json and if isZkEVM is true
+  local IS_ZK_EVM=$(jq -r --arg network "$NETWORK" '.[$network].isZkEVM // false' "$NETWORKS_JSON_FILE_PATH")
```

Comment:
I dont think it's correct to default to false here since this could lead to unexpected behaviour. Would propose to check if the flag was read correctly and throw an error if not, only then proceed.

Reply:
updated